runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -427,44 +427,15 @@ auto LightningSimulator::Measure(QubitIdType wire, std::optional<int32_t> postse
 
     std::vector<double> probs(1U << wires.size());
     DataView<double, 1> buffer_view(probs);
+    auto device_shots = GetDeviceShots();
+    SetDeviceShots(0);
     this->PartialProbs(buffer_view, wires);
```

Comment:
No harm on removing this unnecessary `this` for method calls
```suggestion
    PartialProbs(buffer_view, wires);
```